File: xml-performance/test-drive-xml-main-runner/src/main/java/org/jesperancinha/test/drive/books/XmlTestRunnerOptions.java
package org.jesperancinha.test.drive.books;

import org.kohsuke.args4j.Option;

public class XmlTestRunnerOptions {

	@Option(name = "--book-test-file", aliases = "-b", required = true)
	private String bookTestFile;

	@Option(name = "--number-of-iterations", aliases = "-n", required = true)
	private Integer numberOfIterations;

	@Option(name = "--noop", required = false)
	private boolean noop = false;

	public String getBookTestFile() {
		return bookTestFile;
	}

	public Integer getNumberOfIterations() {
		return numberOfIterations;
	}

	public boolean isNoop() {
		return noop;
	}
}

File: xml-performance/test-drive-xml-main-runner/src/main/kotlin/org/jesperancinha/test/drive/books/XmlTestRunnerOptions.kt
package org.jesperancinha.test.drive.books

import picocli.CommandLine
import picocli.CommandLine.Option


class XmlTestRunnerOptions {
    @Option(name = "--book-test-file", aliases = ["-b"], required = true)
    var bookTestFile: String? = null

    @Option(name = "--number-of-iterations", aliases = ["-n"], required = true)
    var numberOfIterations: Int? = null

    @Option(name = "--noop", required = false)
    var isNoop = false
}